eslint.config.mjs

```diff
@@ -186,6 +186,7 @@ export default [
 			"sonarjs/no-use-of-empty-return-value": "error",
 			"sonarjs/no-nested-template-literals": "warn",
 			"sonarjs/prefer-single-boolean-return": "off",
+			"sonarjs/no-one-iteration-loop": "off",
```

Comment:
if this is due to one of the helper files erroring on it, I resolved that in my PR

Reply:
reason there was not due to the rule itself, but some unnecessary typescript types that their parser didn't handle. Alternatively you can ignore it in the file itself if it remains

Reply:
sounds fair @goga-m , schnorr will be gone by then and others that cause issues might as well be